en-US/firefox/accounts-2018.lang

```diff
@@ -87,11 +100,11 @@ Scan the QR Code to download the Pocket App on your mobile device.
 
 
 ## TAG: accounts_2018_qrcodes
+# String is obsolete. Do not remove.
```

Comment:
This one is obsolete now, but still has a tag... do we remove the tag comment above, or even remove the string entirely since we're falling back to the previously obsolete original?

Reply:
If we are to remove the string, remove the tag too. If *all* the strings sharing the same tag are removed in this update, remove the tag on the top of the page (above `##NOTE` section) as well.

Reply:
Not all of them, just this one. I wasn't sure if I should remove it or leave it marked as obsolete (which is what I did). I'll remove it.